tests/test-suite/test-app/http.ts

```diff
@@ -4,9 +4,25 @@ import * as express from "express";
 import { jsonApiKoa, jsonApiExpress } from "./kurier";
 
 const koaApp = new Koa();
+koaApp.use(async (ctx, next) => {
+  if (ctx.req.url === '/favicon.ico') {
+    ctx.res.statusCode = 404;
+    ctx.res.end();
+  } else {
+    await next();
+  }
```

Comment:
This is just me being picky, but I think we can probably create a compatible middleware for both apps:

```ts
const faviconMiddleware = async (req, res, next) => {
  // ...
};

koaApp.use(async (ctx, next) => faviconMiddleware(ctx.req, ctx.res, next));
expressApp.use(faviconMiddleware);
```

Reply:
hehe this is a funny story. While debugging the test-app in the browser I noticed that breakpoint inside operations get triggered twice in a row, but resource is undefined in the second pass. It took me a while until I figure out it was automatic favicon.ico browser request. Such a waste of time :)
I like the suggestion... will do it.